### PR TITLE
Add spätzle liveticker

### PR DESCRIPTION
**Pull request by leonwind**
- [X] I made sure that the arrays in `routes.php` are still alphabetically sorted. :sparkles: 

As I am a big Spätzle Fan, I always want to know when I can get my favorite Mensa food again!

Very minimal website which would get added: https://leonwind.github.io/spaetzle-ticker/

### CODE REVIEW

**Review by joschahenningsen — APPROVED** [review]
Looks fun :D 

If you want, you can omit the resource heavy web scraping in the browser and utilize the eat-api instead: https://github.com/TUM-Dev/eat-api#api

**Comment by leonwind** [comment]
Thanks for merging so quickly :tada: 

Cool, didn't know there was a public API available! :o 
Will take a look at it if I have some time

Either way, is there no CD anymore? Thought this deploys automatically, or did I break something?
`spaetzle.tum.sexy` is still not redirecting 

If there is no automatic CD, then no rush, just wanted to check in if there is something broken :)

**Comment by joschahenningsen** [comment]
We do have cd in place :thinking: 

@CommanderStorm Do you know why argo didn't update the pod?
image

**Comment by CommanderStorm** [comment]
We don't restart the deployment though.
=> Deployment targeting "latest" stays deployed, even though we have updated latest.

I have a plan on how to refactor this out, but this is an after-exams project

**Comment by CommanderStorm** [comment]
Plan is
- to create a new (private) reopository
- Add https://github.com/TUM-Dev/NavigaTUM/tree/deployment/staging
- on push (this repo):
  update tags in the helm chart to what should be deployed
  See https://github.com/TUM-Dev/NavigaTUM/blob/main/.github/workflows/add-deployment.yaml for a starting point

File: redirect/routes.php [hunk 1]
```diff
@@ -657,10 +657,14 @@ class Route {
         'sp'               => [
             'description' => 'Studienplan B.Sc. Informatik',
             'target'      => 'https://www.cit.tum.de/cit/studium/studiengaenge/bachelor-informatik/studienplan/',
-        ],
+		],
         'sp18'               => [
             'description' => 'Studienplan B.Sc. Informatik beginn WS 18/19',
             'target'      => 'https://www.in.tum.de/fuer-studierende/bachelor-studiengaenge/informatik/studienplan/studienbeginn-wise-201819-bis-sose-2021/',
+		],
+		'spaetzle'		   => [
```
Comment (kordianbruck) [inline redirect/routes.php:665]: The indentation here is seemingly wrong.

**Comment by kordianbruck** [comment]
For now you can recreate (replace) the deployment in argocd and it will restart the pod at the newest latest.

**Pull request merged**